translations/translationtool/src/translationtool.php

```diff
@@ -302,12 +313,32 @@ private function createFakeFileForAppInfo() {
 			$content .= '$l->t(' . $this->escape($string) . ');' . PHP_EOL;
 		}
 		
-		file_put_contents($this->dummyFileName, $content);		
+		file_put_contents($this->fakeAppInfoFile, $content);
+	}
+
+	private function createFakeFileForVueFiles() {
+		$fakeFileContent = '';
+
+		foreach ($this->findTranslatableFiles(['.vue']) as $vueFile) {
+			$vueSource = file_get_contents($vueFile);
+			if ($vueFile === false) {
+				echo "Warning: could not read " . $vueFile . PHP_EOL;
+				continue;
+			}
+			preg_match_all("/t\s*\('([a-zA-Z]+)',\s*'(.+)'/", $vueSource, $singleQuoteMatches);
+			preg_match_all("/t\s*\(\"([a-zA-Z]+)\",\s*\"(.+)\"/", $vueSource, $doubleQuoteMatches);
```

Comment:
can we have `n()` support too?
https://github.com/nextcloud/server/blob/master/apps/updatenotification/src/components/root.vue#L207

Reply:
Whoops, yes! Will look into that!